### PR TITLE
Updated how imports are added.

### PR DESCRIPTION
**Pull request by Osama-Kassem**
This PR adjusts the logic of adding imports, and should fix the bug we had in ua-contracts api-params.go generation.
Also, some of the tests did not compile, I fixed the ones I could. `serve.txt` still does not run, added a TODO to remember that.

### CODE REVIEW

**Review by alnvdl-work — APPROVED** [review]
LGTM, thanks!

File: testdata/serve.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ openapi-httprequest --outputdir server --server api.yaml
 
 # Check that we can build the resulting code.
 # TODO check that it actually runs and behaves as expected.
+# TODO(osama-kassem): This test does not work, fix it.
```
Comment (alnvdl-work) [inline testdata/serve.txt:10]: Isn't this a problem? Wasn't it working before your change?

Please open a ticket on this repo instead, and add a card in JIRA for us not to lose track of it.

Comment (Osama-Kassem) [reply testdata/serve.txt:10]: My change didn't affect it, it was not working.
Will create a ticket & a JIRA card.

**Review by andrei-polukhin — APPROVED** [review]
Thanks for working on this!

**Pull request merged**